build.gradle.kts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.gradle.api.JavaVersion.VERSION_1_8
 import org.gradle.api.tasks.wrapper.Wrapper.DistributionType.ALL
 
 plugins {
```

Comment:
The vert.x plugin already applies `java`

Reply:
Does it apply version 1.8?